recipes/fruit/all/conanfile.py

```diff
@@ -12,36 +18,30 @@ class FruitConan(ConanFile):
     url = "https://github.com/conan-io/conan-center-index"
     homepage = "https://github.com/google/fruit"
     license = "Apache-2.0"
-    topics = ("conan", "fruit", "injection")
+    topics = ("injection", "framework")
     settings = "os", "compiler", "build_type", "arch"
     options = {"shared": [True, False],
                "use_boost": [True, False],
                "fPIC": [True, False]}
     default_options = {"shared": False, "use_boost": True, "fPIC": True}
-    generators = "cmake", "cmake_find_package"
-    exports_sources = ["CMakeLists.txt", "patches/*"]
-    _cmake = None
-
-    @property
-    def _source_subfolder(self):
-        return "source_subfolder"
-
-    @property
-    def _build_subfolder(self):
-        return "build_subfolder"
 
-    def build_requirements(self):
-        if self.options.use_boost:
-            self.build_requires("boost/1.72.0")
+    def export_sources(self):
+        export_conandata_patches(self)
 
     def config_options(self):
         if self.settings.os == "Windows":
             del self.options.fPIC
 
-    def configure(self):
-        if self.options.shared:
-            del self.options.fPIC
+    def layout(self):
+        cmake_layout(self, src_folder="src")
+
+    def requirements(self):
+        if self.options.use_boost:
+            self.requires("boost/1.80.0")
```

Comment:
@uilianries Was there a specific reason for you to update to newest boost?
The pipeline did not find the boost package. I do not yet know why.
When searching manually, I find it.

`conan search 'boost/1.80.0@' -r conancenter -q 'arch=x86_64 AND build_type=Release AND compiler=gcc AND compiler.libcxx=libstdc++11 AND compiler.version=11 AND os=Linux AND shared=False'`

Reply:
gcc 11 might have been missing but this seems to be passing now